diff --git a/firefly_iii_client/model/attachment.py b/firefly_iii_client/model/attachment.py
index 4956bb9..199f5f2 100644
--- a/firefly_iii_client/model/attachment.py
+++ b/firefly_iii_client/model/attachment.py
@@ -99,7 +99,7 @@ class Attachment(ModelNormal):
             'mime': (str,),  # noqa: E501
             'notes': (str, none_type,),  # noqa: E501
             'size': (int,),  # noqa: E501
-            'title': (str,),  # noqa: E501
+            'title': (str, none_type,),  # noqa: E501
             'updated_at': (datetime,),  # noqa: E501
             'upload_url': (str,),  # noqa: E501
         }

